legacy/firmware/fido2/resident_credential.c

```
@@ -0,0 +1,151 @@
+               len - RP_ID_HASH_LENGTH);
+        cred_desc[count].cred_id_len = len - RP_ID_HASH_LENGTH;
+        cred_desc[count].type = PUB_KEY_CRED_PUB_KEY;
+        ctap_authenticate_credential_data(rp_id_hash, &cred_desc[count]);
+        count++;
+      }
+    }
+  }
+  return count;
+}
+
+bool resident_credential_store(const uint8_t *rp_id_hash,
```

cppcheck (2.10-2)

[style] The function 'resident_credential_store' is never used.

(unusedFunction)